app/src/main/java/org/fossasia/pslab/activity/ShowLoggedData.java

```diff
@@ -1,14 +1,17 @@
 package org.fossasia.pslab.activity;
 
 import android.content.Context;
+import android.content.SharedPreferences;
 import android.os.Bundle;
 import android.os.Environment;
+import android.preference.PreferenceManager;
 import android.support.annotation.Nullable;
 import android.support.v7.app.AppCompatActivity;
 import android.support.v7.widget.DefaultItemAnimator;
 import android.support.v7.widget.LinearLayoutManager;
 import android.support.v7.widget.RecyclerView;
 import android.support.v7.widget.Toolbar;
+import android.util.Log;
```

Comment:
Hi @viveksb007 ! The rest looks ok! :+1: Nice work! I guess this import is unused now! :)